python/src/puml/generate_plantuml_diagrams_impl.py

```diff
@@ -20,10 +20,9 @@
 from aac.in_out.writer import write_file
 from aac.in_out.parser._parse_source import parse
 
+from .helpers.component_helpers import _model_sort
 from .helpers.sequence_helpers import _get_use_case_participants, _get_use_case_steps
 
```

Comment:
Too far, you removed the `plugin-name` reference, it was the only part of the string constants that needed to remain.

Reply:
My bad, re-added

Reply:
My bad, re-added